### ISSUE TITLE
Support for adding examples via GUI

### ISSUE THREAD

**Issue by Devin-Grant** [Apicurio/apicurio-studio#378]
I would be nice if the user is able to add examples without having to switch to Source mode.  It should also show the examples under the description (_italicized_).

**Comment by EricWittmann** [comment]
For which entities?  Examples are already supported in a few places (e.g. Request Body and Response for OpenAPI 3 docs). 

I guess we should allow managing examples everywhere it's possible. :)

**Comment by Devin-Grant** [comment]
I agree with adding examples where it's supported.  I was needed to add examples in the model definition for an existing property.

**Comment by EricWittmann** [comment]
OK thanks for the feedback.
